src/utils/utils.js

```diff
@@ -184,7 +184,7 @@ function updateObj<T: Object>(obj: T, fields: $Shape<T>) : T {
  * @static
  */
 function throttle(func: any, ms: number) {
-  let timeout, _this;
+  let timeout, _this; // eslint-disable-line no-shadow
```

Comment:
I got this error, which I thought it's fine to disable it.

<img width="609" alt="screen shot 2017-02-01 at 12 49 48 am" src="https://cloud.githubusercontent.com/assets/4562118/22475642/91e5cf00-e81a-11e6-86b4-8b46117595bf.png">